src/main/java/mate.academy/Main.java

```diff
@@ -6,12 +6,15 @@
 import mate.academy.service.ProductService;
 
 public class Main {
-
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
-        products.forEach(System.out::println);
+        try {
+            ProductService productService = (ProductService) injector
+                    .getInstance(ProductService.class);
+            List<Product> products = productService.getAllFromFile("products.txt");
```

Comment:
Make sure that the file `products.txt` exists in the expected directory and is accessible by the application. Otherwise, this line will throw an exception.